src/turtle/p3turtle.h

```diff
@@ -188,6 +188,7 @@ class TurtleTunnel
 		TurtlePeerId local_dst ;		// where packets should go. Direction to the destination.
 		uint32_t	time_stamp ;			// last time the tunnel was actually used. Used for cleaning old tunnels.
 		uint32_t transfered_bytes ;	// total bytes transferred in this tunnel.
+		uint64_t total_bytes ;
```

Comment:
Add a comment here telling what total_bytes means. Indeed, just above, the transfered_bytes variable  seems to do exactly the same thing if the name is correct.